### DISCUSSION TITLE
Control the width of the menu cart panel

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21214] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

Currently there is no option to control the width of the menu cart widget panel. Please add a new option to control its width. This is a very important and necessary feature.

### Describe the solution you'd like

New option to adjust the width of the Menu cart panel : 

![menu cart width](https://user-images.githubusercontent.com/88034902/210254978-1beff478-f818-4671-b0f7-b3de8cb80159.png)

**Comment by nicholaszein** [comment]
Hello @Michaelgimii,

thanks for submitting your feature request! 🙏 

Allow us to take some time to review your request. We will get back to you with feedback as soon as possible.

Best regards
